spec/rails_helper.rb

```diff
@@ -8,6 +8,16 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+Capybara.register_driver :selenium do |app|
+  options = Selenium::WebDriver::Chrome::Options.new(args: %w[no-sandbox headless disable-gpu])
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+end
```

Comment:
Set up for getting javascript to work in rspec

Reply:
Glad you were able to figure out testing! This looks complicated.